package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
-    "prettify": "prettier --write ."
+    "prettify": "prettier --write .",
+    "stylelint-check": "stylelint-config-prettier-check"
```

Comment:
There is a misunderstanding with this script. You said:

> • npm run stylelint-check to check if it is working

This script doesn't check if you have style issues. You can verify it by adding an invalid css property and run that script.

A script to check for style issues is missing.